Document PHP FPM Healthcheck (#79)
usabilla/php-docker-template: README.md

## Using and extending

### PHP FPM healthcheck

> This image ships with the [php-fpm-healthcheck](https://github.com/renatomefi/php-fpm-healthcheck) which allows you to healthcheck FPM apart from the Nginx setup, providing more compatibility with the single process Docker container.

Reviewer: ```suggestion
This image ships with the [php-fpm-healthcheck](https://github.com/renatomefi/php-fpm-healthcheck) which allows you to healthcheck FPM apart from the Nginx setup, providing more compatibility with a single process Docker container.
```

Author: `The` was used intentionally here, it's a reference to https://cloud.google.com/solutions/best-practices-for-building-containers#package_a_single_app_per_container

> This healthcheck provides diverse metrics to be watched upon and can be fitted accordingly to your needs, more information in how to use it in the [official documentation](https://github.com/renatomefi/php-fpm-healthcheck#a-php-fpm-health-check-script).

Reviewer: ```suggestion
This healthcheck provides diverse metrics to watch and can be configured according to your needs. More information on how to use it can be found in the [official documentation](https://github.com/renatomefi/php-fpm-healthcheck#a-php-fpm-health-check-script).
```

> This healthcheck provides diverse metrics to be watched upon and can be fitted accordingly to your needs, more information in how to use it in the [official documentation](https://github.com/renatomefi/php-fpm-healthcheck#a-php-fpm-health-check-script).

Reviewer: Should we include some examples of these metrics?

Author: These are all documented in the repo, I want to avoid duplication!
And we might be able to contribute back there? hehe

> The healthcheck can be found in the container `$PATH` as executable:

Reviewer: ```suggestion
The healthcheck can be found in the container `$PATH` as an executable:
```

```console
$ php-fpm-healthcheck
$ echo $?
0
```

## Basic usage

Simply use the images as base of the application's `Dockerfile` and apply the necessary changes.

```Dockerfile
# syntax=docker/dockerfile:1.0.0-experimental

FROM usabillabv/php:7.3-fpm-alpine3.9
```

> In usual cases it might not be necessary to extend the nginx images, unless you desired extend it's behavior by for instance serving static files.

Reviewer 2: ```suggestion
In usual cases it might not be necessary to extend the nginx images, unless you desire to extend it's behavior by for instance serving static files.
```
